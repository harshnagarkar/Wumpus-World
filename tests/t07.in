aed-hcbfg
-abcdefgh
id
30
