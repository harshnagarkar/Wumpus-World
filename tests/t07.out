 a e d
 - h c
 b f g

 a e d
 h - c
 b f g

 a e d
 h f c
 b - g

 a e d
 h f c
 - b g

 a e d
 - f c
 h b g

 a e d
 f - c
 h b g

 a e d
 f c -
 h b g

 a e d
 f c g
 h b -

 a e d
 f c g
 h - b

 a e d
 f c g
 - h b

 a e d
 - c g
 f h b

 a e d
 c - g
 f h b

 a e d
 c g -
 f h b

 a e d
 c g b
 f h -

 a e d
 c g b
 f - h

 a e d
 c - b
 f g h

 a - d
 c e b
 f g h

 a d -
 c e b
 f g h

 a d b
 c e -
 f g h

 a d b
 c - e
 f g h

 a - b
 c d e
 f g h

 - a b
 c d e
 f g h

Total nodes: 828183, max depth is: 21
